lib/language-dataset.js

```diff
@@ -275,15 +278,15 @@ export default class LanguageDataset {
         }
       }
     }
-
     return result
   }
 
   hasMatchingForms (partOfSpeech, inflection) {
     if (this.pos.has(partOfSpeech)) {
       let inflectionSet = this.pos.get(partOfSpeech)
+
       if (inflectionSet.types.has(Form)) {
-        return inflectionSet.types.get(Form).items.find(item => this.matcher([inflection], item)) !== undefined
+        return inflectionSet.types.get(Form).items.find((item) => this.matcher([inflection], item)) !== undefined
```

Comment:
@kirlat,  If to be honest I am not quitely agree with you.

Because there is only one case when there is no need in parentheses - when there is only one argument. And If appeling to better consistency than it is better to alway use them.

But some developers like to remove them for one argument. 

And there are a lot of discussion on this case in arbnb group - for example I have found this
https://github.com/airbnb/javascript/pull/276

I could remove if you really think that it is better - but i think that depends only on someone preferences.

Reply:
Agree that it's a matter of one's preference and we can go back and forth discussing what is better. @balmas, what do you think? Do we ever need to care about this?

If we decide to use parentheses should we then change it in the rest of the code, because in most of the places we don't have them? Or is it not so important? What do you think?